bin/addons/godot_whisper/capture_stream_to_text.gd

```diff
@@ -129,7 +129,7 @@ func _add_timer():
 	var timer_node = Timer.new()
 	timer_node.one_shot = false
 	timer_node.autostart = true
-	timer_node.wait_time = 1
+	timer_node.wait_time = 0.5
```

Comment:
We should make this configurable from the node. I think 1s was good enough for my use case for eg.